Signal/src/view controllers/FullImageViewController.m

```diff
@@ -274,21 +306,30 @@ - (void)presentFromViewController:(UIViewController *)viewController {
                      animated:NO
                    completion:^{
                        UIWindow *window = [UIApplication sharedApplication].keyWindow;
+                       // During the presentation animation, we want to seamlessly animate the image
```

Comment:
Decided to add some comments around the presentation animation which is complicated.